src/com/mateacademy/botany/Chamomiles.java

```diff
@@ -0,0 +1,9 @@
+package com.mateacademy.botany;
+
+public class Chamomiles extends Flowers {
+    private static final int price = 70;
```

Comment:
better dont make it static, because in theory different chamomiles can have different price. and better move this field to its parent class